### PR TITLE
Add first version with single droplet barcode plot

### PR DESCRIPTION
**Pull request by pinin4fjords**
The Gene Expression Team at the EBI uses a number of plots internally while analysing data for Single-cell Expression Atlas. We'd like to start putting them in a Bioconda package and Biocontainer for simpler distribution and re-use. Only one script so far (enhanced barcode plotting as discussed here: https://github.com/COMBINE-lab/salmon/issues/362#issuecomment-490160480), more to come.

----

> Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences**
      (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

> Everyone has access to the following BiocondaBot commands, which can be given in a comment:
>
>  * `@BiocondaBot please update` will cause the BiocondaBot to merge the master branch into a PR
>  * `@BiocondaBot please add label` will add the `please review & merge` label.
>  * `@BiocondaBot please fetch artifacts` will post links to packages and docker containers built by the CI system. You can use this to test packages locally before merging.
>
> For members of the Bioconda project, the following command is also available:
>
>  * `@BiocondaBot please merge` will cause packages/containers to be uploaded and a PR merged. Someone must approve a PR first! This has the benefit of not wasting CI build time required by manually merging PRs.
>
> If you have questions, please post them in gitter or ping `@bioconda/core` in a comment (if you are not able to directly ping `@bioconda/core` then the bot will repost your comment and enable pinging).

### CODE REVIEW

File: recipes/scxa-plots/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  sha256: 1aa824fcbb981d63ca1e329c3731c2b52d8d76777b082d6011ce668e1abec829 
+
+build:
+  number: 0
+  noarch: generic
+
+requirements:
+    run:
+        - bioconductor-dropletutils
+        - openblas
```
Comment (bgruening) [inline recipes/scxa-plots/meta.yaml]: why do you need openblas as runtime depdendeny?

Comment (pinin4fjords) [reply recipes/scxa-plots/meta.yaml]: It was required when we first build this tool (I don't recall the error that prompted it to be added). Happy to take it out and reinstate later if required.

**Comment by pinin4fjords** [comment]
@BiocondaBot please fetch artifacts

**Comment by pinin4fjords** [comment]
@BiocondaBot please fetch artifacts

**Comment by pinin4fjords** [comment]
Damn- forgot to merge with the bot...

**Pull request merged**